### PR TITLE
Add verbose option to TVAE

### LINKED ISSUE #307: Tracking and Saving TVAE Loss Values
### Environment details

* CTGAN version: 0.7.3
* Python version: 3.9.6
* Operating System: macOS

### Problem description

When fitting CTGAN, I can capture the loss values in the output variable. However, the same cannot be implemented when fitting the TVAE. How can I track and store the loss values to see if the model is able to learn the distribution of the data over the selected number of epochs?

### PR DESCRIPTION
**Pull request by jjmarks**
Resolves #307.

### CODE REVIEW

**Comment by npatki** [comment]
Hi @jjmarks! Thanks for your interest in contributing to the SDV software. In order to review and approve your code changes, we require that you read and sign our new Contributor License Agreement (CLA).

To request a CLA, please fill out the required information in this form: https://bit.ly/sdv-cla-form

Once we receive your submission, we'll get back to you with more details. Thanks, and feel free to respond here if you have any questions.

**Comment by npatki** [comment]
Hi @jjmarks, this is just a confirmation that we've received your signed CLA. A member of the SDV team will now be able to review/merge your PR. Thanks!

File: ctgan/synthesizers/tvae.py [hunk 1]
```diff
@@ -110,6 +110,7 @@ def __init__(
         decompress_dims=(128, 128),
         l2scale=1e-5,
         batch_size=500,
+        verbose=False,
```
Comment (amontanez24) [inline ctgan/synthesizers/tvae.py:113]: can we add this parameter to the end? Just in case anybody is passing in all the parameters unnamed, we don't want the wrong values being passed to the wrong parameters

File: ctgan/synthesizers/tvae.py [hunk 2]
```diff
@@ -176,6 +178,12 @@ def fit(self, train_data, discrete_columns=()):
                 optimizerAE.step()
                 self.decoder.sigma.data.clamp_(0.01, 1.0)
 
+            if self.verbose:
+                print(f'Epoch {i+1}, Loss: {loss.detach().cpu(): .4f},',  # noqa: T001
+                      f' Rec loss: {loss_1.detach().cpu(): .4f},',
+                      f' KL loss: {loss_2.detach().cpu(): .4f}',
+                      flush=True)
```
Comment (amontanez24) [inline ctgan/synthesizers/tvae.py:185]: would you mind adding unit tests for this case? Just making sure the print out only happens when verbose is true and that the output is as expected

File: ctgan/synthesizers/tvae.py (hunk 1 above)
Comment (jjmarks) [reply ctgan/synthesizers/tvae.py:113]: Can be passed to the end, although this ordering is consistent with CTGAN class (https://github.com/sdv-dev/CTGAN/blob/2848a42651c440a01a987a27658722069c7bb9b1/ctgan/synthesizers/ctgan.py#L145). Would it be better to keep the two orderings consistent?

File: ctgan/synthesizers/tvae.py (hunk 2 above)
Comment (jjmarks) [reply ctgan/synthesizers/tvae.py:185]: Sure. Are there any active unit tests run for TVAE as a template? I see `test_tvae.py` but it looks like only placeholder comments.

File: ctgan/synthesizers/tvae.py (hunk 1 above)
Comment (amontanez24) [reply ctgan/synthesizers/tvae.py:113]: Hmm I guess it's fine to keep consistent.

File: ctgan/synthesizers/tvae.py (hunk 2 above)
Comment (amontanez24) [reply ctgan/synthesizers/tvae.py:185]: Unfortunately I don't think there are, although I think a PR with some should be coming in shortly. I would use the tests in this file as an [example](https://github.com/sdv-dev/CTGAN/blob/master/tests/unit/synthesizer/test_ctgan.py). I know the tests don't exist for the CTGAN verbose parameter, but we're trying to improve test coverage.

I think you can do something similar to [this](https://github.com/sdv-dev/SDV/blob/067c4e3e40631083a283e9bf8cafa525228ce9b2/tests/unit/multi_table/test_base.py#L86-L96). Mock print and make sure it gets called with the correct string but only if verbose is True.
